src/plugins/azgenerator/CodeModelAzImpl.ts

```diff
@@ -1825,6 +1825,8 @@ export class CodeModelCliImpl implements CodeModelAz {
                 // else {
                 //     param_name = "name";
                 // }
+            } else if (param_name.endsWith('_')) {
```

Comment:
Why we need logic here?

Reply:
if not, codegen will have this issue #476 .
`id` will be generate to `id_` or `id-`